requests/configuration/set.go

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Technologies Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ *******************************************************************************/
+
+package configuration
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/edgexfoundry/go-mod-core-contracts/models"
+)
+
+type Key string
+
+/*
+ * SetConfigRequest is for SMA processing of incoming (PUT) requests to Set Config.
+ *
+ * SetConfigRequest and SetConfigResponse struct each follows
+ */
+type SetConfigRequest struct {
+	Key         string `bson:"key" json:"key,omitempty"`
+	Value       string `bson:"value" json:"value,omitempty"`
+	isValidated bool   // internal member used for validation check
```

Comment:
For each property can you add a comment to the right briefly describing each one for GoDoc purposes?